src/screens/contacts/contacts.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { View, Text, SafeAreaView, Button } from 'react-native';
+import { ListItem, Avatar, Overlay, Input } from 'react-native-elements';
+import { FlatList, ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
```

Comment:
Let's use FlatList, ScrollView, TouchableOpacity from react-native instead of react-native-gesture-handler.